src/imutils.jl

```diff
@@ -28,5 +35,10 @@ end
     zeroonenorm(x::AbstractArray)
 
 Normalizes the elements of the array to values between `0` and `1`.
+
```

Comment:
Shouldn't you then remove this workaround and upgrade Zygote in your Project.toml?

Reply:
The main issue is there are some blocking issues on Zygote's end that prevent me from updating it. Even if it is easy to workaround on the end of RayTracer, those changes cause segfault in case of Duckietown.